notebooks/Annotation and Continuous Learning Overview.ipynb

```diff
@@ -72,6 +72,11 @@
     "dicom_client_id = \"<DICOMWeb user ID>\"    # If Authentication is enabled, then provide username, otherwise fill it with the default username \"orthanc\"\n",
     "dicom_client_secret = \"<DICOMWeb secret>\" # If Authentication is enabled, then provide password, otherwise fill it with the default password \"orthanc\"\n",
     "\n",
+    "# The cloud storage type used in this notebook. Currently only support `aws` and `azure`.\n",
+    "cloud_type = \"azure\" # cloud storage provider: aws or azure\n",
+    "cloud_account = \"account_name\" # if cloud_type == \"aws\"  should be \"access_key\"\n",
```

Comment:
Hi @binliunls , there are some warning in the current MONAI tests. Can you take a look at these?

```
Unable to parse line 7 'cloud_account = "account_name" # if cloud_type == "aws"  should be "access_key"'.
Unable to parse line 8 'cloud_secret = "access_key" # if cloud_type == "aws" should be "secret_key"'.
```